### PR TITLE
Update Resizer.php

### PR DESCRIPTION
**Pull request by PubliAlex**
Allow to add watermark to images using the intervention image insert() method.

Still need to write the right validation rule (I suck with regex)

### CODE REVIEW

**Comment by bradietilley** [comment]
@PubliAlex Seeing as a URL regex is quite a mess (a [good URL regex](https://mathiasbynens.be/demo/url-regex) will take upwards of 250-500 chars), I would simply validate it as the following: `^(.+),(\d+),(\d+)$` - this allows literally anything as the URL, but at least all valid URLs will actually work. In my opinion it's up to the developer to ensure that the URL is valid before passing it through this function.

If you add this regex I'll go ahead and merge this in :)

File: classes/Resizer.php [hunk 1]
```diff
@@ -795,6 +799,14 @@ public function modify()
                     list($r, $g, $b) = explode(',', $value);
                     $this->im->colorize($r, $g, $b);
                     break;
+                case 'insert':
+                    $exp = explode(',', $value);
+                    $path = $exp[0];
+                    $position = (isset($exp[1]))? $exp[1] : null;
+                    $x = (isset($exp[2]))? $exp[2] : null;
+                    $y = (isset($exp[3]))? $exp[3] : null;
```
Comment (bennothommo) [inline classes/Resizer.php]: @PubliAlex please add a space between the closing brackets and the question marks in the lines above.

Comment (PubliAlex) [reply classes/Resizer.php]: Regex and space added :)

**Comment by bradietilley** [comment]
Added to [2.0.2](https://octobercms.com/plugin/abwebdevelopers-imageresize)

**Pull request merged**